src/phone-number-input/phone-number-input.tsx

```diff
@@ -133,7 +134,7 @@ export const PhoneNumberInput = ({
                     onSelectOption: handleSelectOption,
                     onHideOptions: onHideOptions,
                     onShowOptions: onShowOptions,
-                } as ListAddon<CountryValue, string>,
+                } as unknown as ListAddon<CountryValue, string>,
```

Comment:
didn't realise `readonly` was problematic. can revert the change to avoid casting to `unknown`

Reply:
Actually i see value in setting readonly. But ya this is the workaround. I have no comments/preference. I'll just keep it simple